tests/containers/image.pm

```diff
@@ -27,20 +27,13 @@ sub run {
     # We may test either one specific image VERSION or comma-separated CONTAINER_IMAGE_VERSIONS
     my $versions = get_var('CONTAINER_IMAGE_VERSIONS', get_required_var('VERSION'));
     for my $version (split(/,/, $versions)) {
-        my $images_to_test;
-        # Get array of single image from CONTAINER_IMAGES_TO_TEST or from get_suse_container_urls()
-        unless ($images_to_test = get_container_url_from_var('CONTAINER_IMAGE_TO_TEST')) {
-            my ($untested_images, $released_images) = get_suse_container_urls(version => $version);
-            $images_to_test = check_var('CONTAINERS_UNTESTED_IMAGES', '1') ? $untested_images : $released_images;
-        }
-
-        for my $iname (@{$images_to_test}) {
-            record_info "IMAGE", "Testing image: $iname";
-            test_container_image(image => $iname, runtime => $engine);
-            test_rpm_db_backend(image => $iname, runtime => $engine);
-            my $beta = $version eq get_var('VERSION') ? get_var('BETA', 0) : 0;
-            test_opensuse_based_image(image => $iname, runtime => $engine, version => $version, beta => $beta) unless ($iname =~ /bci/);
-        }
+        my $image = get_image_url();
```

Comment:
is this module something which is scheduled in both `released`, `unreleased`? With current approach you will have always `released=1`. i think this breaks for instance https://openqa.suse.de/tests/8312796#step/image_docker/46

Reply:
Good catch, thank you :smile: